### PR TITLE
Update BulkCopy docs for asynchronous support

### PR DESCRIPTION
**Pull request by Shane32**
See https://github.com/linq2db/linq2db/pull/2314

### CODE REVIEW

File: source/articles/sql/Bulk-Copy.md [hunk 1]
```diff
@@ -44,19 +53,21 @@ SQLite               |   Yes    |     Yes      |        No        | MultipleRows
...
-- `inet` vs `cidr`. If you use `NpgsqlInet` type for mapping column, it could be mapped to both `inet` and 'cidr' types. There is no default mapping for this type, so you should explicitly specify it using `DbType = "inet"` or `DbType = "cidr"` hint. Also for `inet` you can use `IPAddress` which will be mapped to `inet` type.
-- `macaddr` vs `macaddr8`. Both types could be mapped to the same `PhysicalAddress`/`String` types, so you should explicitly specify column type using `DbType = "macaddr"` or `DbType = "macaddr8"` hints. Even if you use provider version without `macaddr8` support, you should specify hint or it will break after provider update to newer version.
-- `date` type. You should use `NpgsqlDate` type in mapping or specify `DataType = DataType.Date` or `DbType = "date"` hints.
-- `time with time zone` type needs `DbType = "time with time zone"` hint.
+- `decimal`/`numeric` vs `money`. Those are two different types, mapped to `System.Decimal`. Default mappings will use `numeric` type, so if your column is the `money` type, you should type it in mappings using `DataType = DataType.Money` or `DbType = "money"` hints.
```
Comment (to11mtm) [inline source/articles/sql/Bulk-Copy.md:64]: `Default mappings will use `numeric` type `

to

`Default mappings will use the `numeric` type`

Comment (Shane32) [reply source/articles/sql/Bulk-Copy.md]: Done

**Comment by Shane32** [comment]
> Thanks! could you add information which providers support async bulk copy (provider specific)

I added this below the table:

> Note that when using the provider-specific insert method, only MySql, PostgreSQL, SAP HANA, and Microsoft SQL Server support asynchronous operation; other providers will silently fall back to a synchronous operation.

If you think it should instead be embedded in the table, either as an additional column, or as notes in the ProviderSpecific column, please let me know.

**Comment by Shane32** [comment]
Another idea: I could replace "Yes" with "Synchronous only" or "Yes; synchronous only" for the IBM and Oracle providers.

**Comment by MaceWindu** [comment]
> I added this below the table

this is fine

**Pull request merged**